utils/build-script-impl

```diff
@@ -2319,6 +2319,7 @@ for host in "${ALL_HOSTS[@]}"; do
                     -DCMAKE_CXX_COMPILER:PATH="${CLANG_BIN}/clang++"
                     -DCMAKE_SWIFT_COMPILER:PATH="${SWIFTC_BIN}"
                     -DCMAKE_Swift_COMPILER:PATH="${SWIFTC_BIN}"
+                    -DCMAKE_Swift_FLAGS:PATH="-module-cache-path \"${module_cache}\""
```

Comment:
```suggestion
                    -DCMAKE_Swift_FLAGS:STRING="-module-cache-path \"${module_cache}\""
```
I think it should be a `STRING` type not `PATH` because you have `-module-cache-path` flag.